source/daq/event_builder.cc

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
Overall I think the event builder logic looks good.  I just had the one question down around line 169.